contribs/gmf/examples/editfeatureselector.js

```diff
@@ -120,14 +130,36 @@ app.MainController = function($scope, gmfThemes, gmfUser, ngeoToolActivateMgr) {
   });
 
   /**
+   * @type {Object.<ol.geom.GeometryType, Array.<ol.style.Style>>}
+   * @private
+   */
+  this.editingStyles_ = ngeo.style.createDefaultEditingStyles();
+
+  /**
    * @type {ol.layer.Vector}
    * @export
    */
   this.vectorLayer = new ol.layer.Vector({
     source: new ol.source.Vector({
       wrapX: false,
       features: new ol.Collection()
-    })
+    }),
+    style: function(feature, resolution) {
+      // (1) Use the geometry collection style, which works fine for any
+      //     geometry types
+      var styles = [].concat(this.editingStyles_[
+        /** @type {ol.geom.GeometryType<string>} */ ('GeometryCollection')
+      ]);
```

Comment:
`ol.style.createDefaultEditingStyles` is not marked as an `@api` method.  Therefore, in the minimized example, it's not defined and it doesn't work.  That's why I added a small "wrapper" around it in, i.e.: `ngeo.style.createDefaultEditingStyles`.  I agree that for the purpose of an example, this is not that good.  Creating the style directly would be simpler.  I'll do that.